presto-kafka/src/test/java/io/prestosql/plugin/kafka/KafkaRawQueryRunner.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kafka;
+
+import com.google.common.collect.ImmutableMap;
+import io.airlift.log.Logger;
+import io.airlift.log.Logging;
+import io.prestosql.Session;
+import io.prestosql.testing.DistributedQueryRunner;
+
+import java.util.Map;
+
+import static io.prestosql.testing.TestingSession.testSessionBuilder;
+
+public final class KafkaRawQueryRunner
```

Comment:
why do you need another kafka query runner? Maybe the existing one should accept some parameters instead?

Reply:
This should be removed, I was using it to test against a local kafka + schema registry with deeply nested avro types. I'll add tests.